extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SimpleScheduler.java

```diff
@@ -695,6 +706,13 @@ public boolean isBlocking() {
             return null;
         }
 
+        private boolean isOTelTracingEnabled() {
+            return ConfigProvider.getConfig().getValue("quarkus.otel.enabled", Boolean.class);
+        }
+
+        private boolean isOTelExtensionAvailable() {
+            return CDI.current().select(io.opentelemetry.api.OpenTelemetry.class).isResolvable();
```

Comment:
I would try to avoid using `CDI.current().select(OpenTelemetry.class)` at all. In this particular case, I would just add the `Instance<OpenTelemetry>` injection point in the `SimpleScheduler` and use it here.